test/Microsoft.AspNetCore.Razor.Test/TestFiles/CodeGenerator/Output/AttributeTargetingTagHelpers.cs

```diff
@@ -40,7 +41,7 @@ public AttributeTargetingTagHelpers()
                     WriteLiteral("Hello");
                     Instrumentation.EndContext();
                 }
-                , StartTagHelperWritingScope, EndTagHelperWritingScope);
+                );
```

Comment:
Ahh, that logic is baked into our scope writer. Going to leave, too much churn for small benefit :smile: